package.json

```diff
@@ -65,77 +65,95 @@
                 "icon": "$(refresh)"
             },
             {
+                "category": "MTA",
```

Comment:
Don't know if we want all of these but it was hard to distinguish these from core functionality.